src/quantities/definitions.js

```diff
@@ -161,6 +161,7 @@ export var UNITS = {
   /* charge */
   "<coulomb>" :  [["C","coulomb","Coulomb"], 1.0, "charge", ["<ampere>","<second>"]],
   "<Ah>" :  [["Ah"], 3600, "charge", ["<ampere>","<second>"]],
+  "<elementary-charge>" :  [["e"], 1.602176634e-19, "charge", ["<ampere>","<second>"]],
```

Comment:
@maasencioh I think you should add unit tests to check the feature and verify it does not conflict with scientific notation for example: 

```js
var qty = new Qty("3e8 m/s");

expect(qty.scalar).toBe(300000000);
expect(qty.numerator).toEqual(["<meter>"]);
expect(qty.denominator).toEqual(["<second>"]);
```